src/components/VideoPlayerStatsTable.vue

```diff
@@ -217,7 +223,8 @@ export default {
   computed: {
     ...mapState('Controls', [
       'isMobile',
-      'isSplittedView'
+      'isSplittedView',
+      'castOptions'
```

Comment:
where's this coming from?

Reply:
The controls state has a property called 'castOptions', which is an object that gets an 'isLoading' property and a 'streamId" property, that is set when the viewer opens. The CastOptions gets all the options configured from the web viewer code.

![image](https://github.com/user-attachments/assets/0fb08223-512d-4684-9a55-75fffc9f78c8)

Reply:
CastOptions is set in the main VideoPlayerContainer.vue upon it being mounted. This gives you the streamId, token, and loading on the castOptions object which is in the controls.js store state.
![image](https://github.com/user-attachments/assets/8beffa97-5653-4faa-9887-b6a27e709148)

Reply:
Ok - @Santiago-Souto can you pls review this PR? :) 

I will rely on you as you're more close to the viewer than me :)

Reply:
Even though castOptions has the streamId variable to be shown, it is used for google casting cases, I don't think it is the best approach. You can take the streamId from the viewer params store, by mapping the stats of `Params` and extracting the `state.viewer` variable, and from there, the streamId

Reply:
I originally did it this way and tried castOptions because I thought it would be a good way to reuse the code already there. Thanks for the confirmation that my initial way was correct. I will make the change now